eo-maven-plugin/src/main/java/org/eolang/maven/RegisterMojo.java

```diff
@@ -137,7 +137,15 @@ public void exec() throws IOException {
                 .withSource(file.toAbsolutePath())
                 .withVersion(ParseMojo.ZERO)
                 .withVer(ParseMojo.ZERO);
+            final String fullname = String.join("|", name, ParseMojo.ZERO);
+            if (this.external != null) {
+                this.externalTojos
+                    .add(fullname)
+                    .withSource(file.toAbsolutePath())
+                    .withVersion(ParseMojo.ZERO);
+            }
             Logger.debug(this, "EO source %s registered", name);
+            Logger.debug(this, "EO source %s registered (external)", fullname);
```

Comment:
@maxonfjvipon It seems that statement should be placed under `if(this.external != null)` scope.